### PR TITLE
Feat uploadsteps

### PR DESCRIPTION
**Pull request by nlvo**

### CODE REVIEW

File: modules/multer.js [hunk 1]
```diff
@@ -12,4 +12,7 @@ const upload = multer({
     storage: storage
 }).single('scanmeds')
 
-module.exports = upload
\ No newline at end of file
+module.exports = upload
+
+// const storage = multer.memoryStorage()
```
Comment (joanpadolina) [inline modules/multer.js]: are you going to use this commented section?

File: routes/scanner.js [hunk 2]
```diff
@@ -3,10 +3,35 @@ const imageToText = require('../modules/createWorker')
...
+                meds: meds[0],
+                image: req.file.originalname
+            })
+        }
+
+    } else {
+        res.render('pages/scan')
+    }
+
+    // res.render('./pages/scan', { text:text, meds:meds[0], image: req.file.originalname})
```
Comment (joanpadolina) [inline routes/scanner.js]: is this line still relevant ? are we going to need this later?

**Review by joanpadolina — COMMENTED** [review]
hi

File: routes/scanner.js (hunk 2 above)
Comment (nlvo) [reply routes/scanner.js]: oh nope, forgot to remove.

File: modules/multer.js (hunk 1 above)
Comment (nlvo) [reply modules/multer.js]: Still contemplating, but will remove it for now. Was thinking about a way to not save images, privacy wise.

Comment (joanpadolina) [reply modules/multer.js]: ah yea I thought so, you can keep it there for later use that's fine 👯

File: routes/scanner.js (hunk 2 above)
Comment (joanpadolina) [reply routes/scanner.js]: oki

**Review by joanpadolina — APPROVED** [review]
yessu goodjob!

**Pull request merged**